tools/ports/sdl2_image.py

```diff
@@ -19,6 +19,9 @@
   'formats': 'A comma separated list of formats (ex: --use-port=sdl2_image:formats=png,jpg)'
 }
 
+SUPPORTED_FORMATS = {'avif', 'bmp', 'gif', 'jpg', 'jxl', 'lbm', 'pcx', 'png',
+                     'pnm', 'qoi', 'svg', 'tga', 'tif', 'webp', 'xcf', 'xpm', 'xv'}
+
```

Comment:
The only two formats we currently support (in process_dependencies) are `png` and `jpg`.   The others are either always-on or not-supported.

Reply:
Oh, I guess we do supply `'-DLOAD_xx` for all formats.. this part lgtm then.

Reply:
But right now you can provide `bmp` for example in `SDL2_IMAGE_FORMATS` and when you do that then the ports code adds `-DLOAD_BMP` to the flags. I am not sure it makes sense to be able to add `foobar` as a format and have it working...

Reply:
We cannot limit the `--use-port=sdl2_image:formats=xxx` to just `png` and `jpg`. That would make no sense at all...

Reply:
Our message responses criss-crossed each other :)

Reply:
It might make sense if all the other formats if we define "formats" as "formats-that-are-not-builtin-already-and-require-and-extra-dependency" which I assume what the original intent was.   This limits significantly the total number of library variants.. which is good thing for testing and binary distribution (e.g. prebuilt and locked cache).

However, this lgtm as is for now.